### ISSUE TITLE
Delete a specific Wishlist.

### ISSUE THREAD

**Issue by reubenabraham** [CSCI-GA-2820-SP23-003/wishlists#12]
**As a** end-user
**I need** a delete en-point
**So that** I can delete a entire wishlist of mine.
   
 ### Details and Assumptions
 * Endpoint signature will look like `DELETE /wishlists/4` which will delete wishlist with `wishlist_id`=4 from the `wishlists` table.
 * Database schema will be set-up to cascade delete, so when a wishlist is deleted, it's corresponding items from the items table are also deleted.
* Assumption: User knows the `wishlist_id` of the wishlist they want to delete. Can be fetched by a read or captured from the http `201` returned when the resource is created.
   
 ### Acceptance Criteria  
   
 ```gherkin
 Given that a `wishlist_id` matches provided wishlist_id
 When the user calls the end-point
 Then cascade delete that wishlist.
```
* The end-point must be unit-tested.
* The end-point must have general logging in place- use Flask's app.logger.

**Comment by reubenabraham** [comment]
@jjrofrano : We assume user X can only delete their own wishlists. Should we check this maybe with a user_id in the http header? Surely we need some authentication of who the user is.

**Comment by rofrano** [comment]
@reubenabraham Yes, this is true but you can add authentication after we have the security lecture and I show you authentication examples. For now, just assume that the caller has the authority to do whatever your API does. Think MVP. Get it working first, then make it better.
